core/storage/exploration/gae_models.py

```diff
@@ -32,6 +34,10 @@
 if MYPY: # pragma: no cover
     from mypy_imports import datastore_services  # pylint: disable=unused-import
 
+# Constants used for generating ids.
+MAX_RETRIES = 10
```

Comment:
just use "base_models.MAX_RETRIES" instead of defining it again

Reply:
Done.